ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export HOME=/tmp
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
+
+# Ensure ignition assets are configured with the correct invoker to track CI jobs.
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+
+export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json
+
+dir=/tmp/installer
+mkdir "${dir}/"
+pushd ${dir}
+cp -t "${dir}" \
+    "${SHARED_DIR}/install-config.yaml"
+
+### Empty the compute pool (optional)
+echo "Emptying the compute pool..."
+python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.load(open(path));
+data["compute"] = [ { "name": "worker", "replicas": 0 } ];
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+### Create manifests
+echo "Creating manifests..."
+openshift-install --dir="${dir}" create manifests &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+if [ $ret -ne 0 ]; then
+  cp "${dir}/.openshift_install.log" "${ARTIFACT_DIR}/.openshift_install.log"
+  exit "$ret"
+fi
+
+### Remove control plane machines
+echo "Removing control plane machines..."
+rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
+
+### Remove compute machinesets (optional)
+echo "Removing compute machinesets..."
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+
+### Make control-plane nodes unschedulable
+echo "Making control-plane nodes unschedulable..."
+python -c '
+import yaml;
+path = "manifests/cluster-scheduler-02-config.yml";
+data = yaml.load(open(path));
+data["spec"]["mastersSchedulable"] = False;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+### Create Ignition configs
+echo "Creating Ignition configs..."
+openshift-install --dir="${dir}" create ignition-configs &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+cp "${dir}/.openshift_install.log" "${ARTIFACT_DIR}/.openshift_install.log"
+
+if [ $ret -ne 0 ]; then
+  exit "$ret"
+fi
+
+cp -t "${SHARED_DIR}" \
+    "${dir}/auth/kubeadmin-password" \
+    "${dir}/auth/kubeconfig" \
+    "${dir}/metadata.json" \
+    ${dir}/*.ign
+
+tar -czf "${SHARED_DIR}/.openshift_install_state.json.tgz" ".openshift_install_state.json"
```

Comment:
Do we need this for something?

Reply:
Yes, otherwise either `wait-for bootstrap-complete` or `wait-for install-complete` prompts for the platform.

Reply:
> Yes, otherwise either `wait-for bootstrap-complete` or `wait-for install-complete` prompts for the platform.

Are we clear on why it prompts?  I would expect that it could get the platform from the cluster via the provided kubeconfig.